src/helpers/email_helpers.ts

```diff
@@ -0,0 +1,54 @@
+import sendgrid from "@sendgrid/mail";
+import { add_audit_record, get_document_by_id } from ".";
+import { isObject } from "lodash";
+import { EmailSettings, Mail } from "../types";
+import { logger } from "../managers";
+
+export const send_email = async (mail: Mail) => {
```

Comment:
1) it should be email and not mail.
2) better to call it EmailDetails, instead of simply email - because it's email is a result of this procedure